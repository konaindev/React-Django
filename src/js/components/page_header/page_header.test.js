import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";
import PageHeader from "./index";

describe("PageHeader", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <PageHeader>header content</PageHeader>
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
