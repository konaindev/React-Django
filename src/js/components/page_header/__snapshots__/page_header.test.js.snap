// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader renders correctly 1`] = `
<div
  className="page-header"
>
  <div
    className="container page-header__inner"
  >
    <a
      href="/dashboard"
    >
      <div
        className="remarkably-logo"
      >
         
      </div>
    </a>
    <nav
      className="page-header__items-outer"
    >
      <ul
        className="page-header__items-inner"
        style={
          Object {
            "gridTemplateColumns": "repeat(1, 1fr)",
          }
        }
      >
        <li
          className="page-header__item"
        >
          header content
        </li>
      </ul>
    </nav>
  </div>
</div>
`;
